apps/meteor/tests/e2e/admin-users.spec.ts

```diff
@@ -1,88 +1,184 @@
-				await page.getByRole('menuitem', { name: 'Deactivate' }).click();
-				await expect(page.getByRole('link', { name: username })).not.toBeVisible();
-
-				await page.getByRole('tab', { name: 'Deactivated' }).click();
-				await expect(page.getByRole('link', { name: username })).toBeVisible();
+				await admin.tabs.users.btnMoreActionsMenu.click();
+				await admin.tabs.users.menuItemDeactivated.click();
+				await expect(admin.getUserRowByUsername(user.data.username)).not.toBeVisible();
+				await admin.tabs.users.tabDeactivated.click();
+				await expect(admin.getUserRowByUsername(user.data.username)).toBeVisible();
 			});
 
 			await test.step('moves from Deactivated to Pending tab', async () => {
-				await page.getByRole('tab', { name: 'Deactivated' }).click();
-				await page.getByRole('button', { name: 'More actions' }).click();
-				await page.getByRole('menuitem', { name: 'Activate' }).click();
-				await expect(page.getByRole('link', { name: username })).not.toBeVisible();
+				await admin.tabs.users.tabDeactivated.click();
+				await admin.tabs.users.btnMoreActionsMenu.click();
+				await admin.tabs.users.menuItemActivate.click();
+				await expect(admin.getUserRowByUsername(user.data.username)).not.toBeVisible();
+				await admin.tabs.users.tabPending.click();
+				await expect(admin.getUserRowByUsername(user.data.username)).toBeVisible();
+			});
+		});
+	});
+	test.describe.serial('Login', () => {
+		let context: BrowserContext;
+		let page: Page;
+		let poRegistration: Registration;
+		let poUtils: Utils;
+
+		test.beforeAll(async ({ browser }) => {
+			context = await browser.newContext();
+			page = await context.newPage();
+			poRegistration = new Registration(page);
+			poUtils = new Utils(page);
+		});
 
-				await page.getByRole('tab', { name: 'Pending' }).click();
-				await expect(page.getByRole('link', { name: username })).toBeVisible();
+		test.afterAll(async () => {
+			await context.close();
+		});
+
+		test('Login as a newly created user and verify its status is active', async () => {
+			await page.goto('/login');
+			await test.step('should log in as the newly created user', async () => {
+				await poRegistration.username.fill(user.data.username);
+				await poRegistration.inputPassword.fill(DEFAULT_USER_CREDENTIALS.password);
+				await poRegistration.btnLogin.click();
+			});
+
+			await test.step('Assert user is logged in', async () => {
+				await expect(poUtils.mainContent).toBeVisible();
+			});
+		});
+	});
+
+	test.describe('User Status Management', () => {
+		test.use({ storageState: Users.admin.state });
+
+		test.beforeEach('Go to /admin/users', async ({ page }) => {
+			admin = new Admin(page);
+			await page.goto('/admin/users');
+		});
+
+		test('After the first login, the user gets listed under the Active tab', async () => {
+			await admin.tabs.users.inputSearch.fill(user.data.username);
+
+			await test.step('is visible in the All tab', async () => {
```

Comment:
maybe we could use the pattern of "should"
should be visible in the All tab
should not..